query-connector/src/app/database-service.ts

```diff
@@ -197,48 +190,12 @@ export async function getVSACValueSet(
   }
 }
 
-/**
- * Translates a VSAC FHIR bundle to our internal ValueSet struct
- * @param fhirValueset - The FHIR ValueSet response from VSAC
- * @param ersdConceptType - The associated clinical concept type from ERSD
- * @returns An object of type InternalValueSet
- */
-export async function translateVSACToInternalValueSet(
```

Comment:
moved this out to its own, non-server component file so that we didn't have to mark it as async to export it